mmseg/utils/util_distribution.py

```diff
@@ -33,6 +33,14 @@ def build_dp(model, device='cuda', dim=0, *args, **kwargs):
         dp_factory['mlu'] = MLUDataParallel
         model = model.mlu()
 
+    if device == 'npu':
```

Comment:
Might add the `elif` statement to the code above

Reply:
I've revised it, thanks